benchmarks/skimage/run-nv-bench-morphology.sh

```diff
@@ -15,9 +15,9 @@ done
 param_filt_float=(remove_small_objects remove_small_holes erosion dilation opening closing white_tophat black_tophat reconstruction)
 param_dt=(float32)
 for shape in "${param_shape[@]}"; do
-    for filt in "${param_filt[@]}"; do
+    for filt in "${param_filt_float[@]}"; do
```

Comment:
I'm assuming that `param_filt_float` is named that way for a reason -- otherwise the variable is unused.  So this particular benchmark has been running on the larger param set defined above on line 3

Reply:
I agree with your read of this, it sure *looks like* this was intended to use `param_filt_float`.

All the changes to this file in the blame ([link](https://github.com/rapidsai/cucim/blame/branch-25.08/benchmarks/skimage/run-nv-bench-morphology.sh)) are from you @grlee77, could you comment on this change?